core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly

```diff
@@ -27,10 +27,10 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
-<l:layout title="Log">
-  <st:include page="sidepanel.jelly" />
+<l:layout title="${%All Jenkins Logs}">
+  <l:breadcrumb title="${%All Jenkins Logs}" />
   <l:main-panel>
-    <h1>${%Jenkins Log}</h1>
+    <h1>${%All Jenkins Logs}</h1>
```

Comment:
Small change just so the titles are consistent, happy to remove if reviewers want.